community.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: page
+title: Community
+permalink: /community/
+redirect_from: /community/forums/
+---
+
+<a name="collaboration"></a>
+### Collaboration
+
+- <a href="https://forums.fusetools.com" target="_blank">Forums</a>
+- <a href="https://slackcommunity.fusetools.com" target="_blank">Slack</a>
```

Comment:
It's unclear to me what this file has to do with the subject of this PR...

Reply:
> Sigh, why _yet_ another PR? Can't you just force-push the branch in the old one? No need to do anything about this now, but please don't create even more PRs for the same thing 😆
Sorry! I dont know how to do it! Thanks for the advise 😆

Reply:
> It's unclear to me what this file has to do with the subject of this PR...
What do you mean? In this PR I propose a new menu section called Community that have inside Articles, links, Packages/Samples, Plugins, etc...

Reply:
Well, It *also* updates some showcases. In the same commit. That's not a good plan IMO. Perhaps you could send the showcase stuff as a separate PR, then?

Reply:
I remove the showcase changes. Only this PR are the new menu section.